pom.xml

```diff
@@ -870,7 +869,7 @@
                 <plugin>
                     <groupId>com.github.cafapi.plugins.docker.versions</groupId>
                     <artifactId>docker-versions-maven-plugin</artifactId>
-                    <version>1.0.0-29</version>
+                    <version>2.0.1-64</version>
```

Comment:
To get CI build passing for my branch - I can bring this out into a new MR if it shouldn't be in this one